static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -105,6 +113,32 @@ function BreadcrumbItem({
           </InspectorWrapper>
         )}
 
+        {'data' in frame && frame.data && 'mutations' in frame.data ? (
+          <Button
+            onClick={() => {
+              openModal(
+                deps => (
+                  <ReplayComparisonModal
+                    replay={replay}
+                    organization={organization}
+                    leftTimestamp={frame.offsetMs}
+                    rightTimestamp={
+                      // @ts-expect-error
+                      frame.data.mutations.next.timestamp -
```

Comment:
Where does `mutations` come from? Don't recall the SDK sending any data

Reply:
ooh, good reminder. It comes from https://github.com/getsentry/sentry/blob/c26fcb45f372af6a152d7c1e93c73f8c0476c7f2/static/app/utils/replays/hydrateBreadcrumbs.tsx#L39-L43 which needs to be updated

Reply:
I guess i can refactor things a bit more to make it prod ready

Reply:
scott got this spot, and some others 👍